packages/beacon-node/test/e2e/chain/stateCache/nHistoricalStates.test.ts

```diff
@@ -236,17 +238,17 @@ describe(
         maxCPStateEpochsInMemory: 1,
         // reload CP state epoch 2 (slot = 16)
         reloadCount: 1,
-        // 1 cp state for epoch 0 1, 2 CP states for epoch 2, 1 cp state for epoch 3
-        persistCount: 5,
+        // {root0, epoch: 0} {root8, epoch: 1} {root16, epoch: 2} {root15, epoch: 2} {root23, epoch: 3} {root24, epoch: 3} {root15, epoch: 3}
+        persistCount: 7,
         // epoch 4, one for Current Root Checkpoint State and one for Previous Root Checkpoint State
         numStatesInMemory: 2,
-        // chain is not finalized, epoch 4 is in-memory so CP state at epoch 0 1 2 3 are persisted, epoch 2 has 2 CP states
-        numStatesPersisted: 5,
+        // chain is not finalized, so same number to persistCount
+        numStatesPersisted: 7,
         // epoch 4
         numEpochsInMemory: 1,
         // chain is not finalized, epoch 4 is in-memory so CP state at epoch 0 1 2 3 are persisted
         numEpochsPersisted: 4,
-        // chain is NOT finalized end of test
+        // chain is NOT finalized end of test,
```

Comment:
are you trying to add a line after comma?

Reply:
removed